packages/aws-cdk-lib/aws-sns/lib/topic.ts

```diff
@@ -173,6 +184,7 @@ export class Topic extends TopicBase {
   public readonly topicName: string;
   public readonly contentBasedDeduplication: boolean;
   public readonly fifo: boolean;
+  public readonly signatureVersion?: string;
```

Comment:
Do we need the public variable?

Reply:
What's the standard policy here? Prefer not to add it?  This means take it out of `topic-base.ts` as well right ?

Reply:
I got rid of the public variable.